nsinit/exec.go

```diff
@@ -51,7 +45,7 @@ func execAction(context *cli.Context) {
 	}
 	process := &libcontainer.Process{
 		Args:   context.Args(),
-		Env:    context.StringSlice("env"),
+		Env:    append(os.Environ(), context.StringSlice("env")...),
```

Comment:
No issues really except that there won't be an easy way to exactly control the process's env.
So, if this is the default, maybe a --disable-env-inherit flag could retain the old behavior or --inherit-os-env to enable this.